GooglePlayPlugins/com.google.play.billing/Samples/TrivialKart/scripts/GameManager.cs

```diff
@@ -50,4 +59,63 @@ public bool IsInPlayPage()
     {
         return playPageCanvas.activeInHierarchy;
     }
+
+    private void PrintCarJson()
+    {
+        sedan = new Car("sedan", 500, 0, false, true);
+       
+        print(JsonUtility.ToJson(sedan));
```

Comment:
What's this method used for?

Reply:
It is just for test purpose, I will delete it